### PR TITLE
Create Typescript file

### PR DESCRIPTION
**Pull request by pablogdcr**

### CODE REVIEW

**Comment by IlliaAgility** [comment]
All is fine! You can merge it

**Comment by silenzium** [comment]
can someone with access merge this please? :)

File: src/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Middleware } from 'redux';
+
+export interface DynamicMiddlewares {
+  enhancer: Middlware,
```
Comment (silenzium) [inline src/index.d.ts:4]: this should be `Middleware` instead of `Middlware`.

**Review by silenzium — CHANGES_REQUESTED** [review]

**Comment by pofigizm** [comment]
Good job
Thank you

**Comment by pofigizm** [comment]
Sorry for waiting

**Pull request merged**